demo/index.html

```diff
@@ -142,6 +142,49 @@
     }());
   </script>
 
+  <p>Here's a pie chart listening for `onmouseover`:</p>
+
+  <style>
+    #mouseover-demo {
+      position: relative;
+      height: 300px;
+    }
+
+    #mouseover-chart {
+      float: left;
+    }
+
+    #mouseover-display {
+      display: inline-block;
+      position: relative;
+      top: 50%;
+    }
+  </style>
+  <div id="mouseover-demo">
+    <google-chart
+      id="mouseover-chart"
+      type="pie"
+      options='{"title": "Distribution of days in 2001H1"}'
+      cols='[{"label": "Month", "type": "string"},{"label": "Days", "type": "number"}]'
+      rows='[ ["Jan", 31],["Feb", 28],["Mar", 31],["Apr", 30],["May", 31],["Jun", 30] ]'
+      events='["onmouseover"]'>
```

Comment:
I see. 30 feels like too many. Opting in will also be better for perf. I was only thinking it would be nice to have similar syntax for each component.